Makefile

```diff
@@ -12,8 +12,8 @@ PROJECT_ID = catalogue_api
 SBT_APPS = search
 SBT_NO_DOCKER_APPS = items requests
 
-SBT_DOCKER_LIBRARIES    =
-SBT_NO_DOCKER_LIBRARIES = display stacks search_common
+SBT_DOCKER_LIBRARIES    = stacks
```

Comment:
What does this do / change?

Reply:
It's because the `stacks` library has a Docker Compose file – so the tests will treat it a bit differently, and start it first.

That used to be the case, anyway…